src/util.ts

```diff
@@ -88,3 +88,16 @@ export const unique = () => {
     return true;
   };
 };
+
+export async function enableCorepack(input: string): Promise<void> {
+  const corepackArgs = ['enable'];
+  if (input.length > 0 && input !== 'false') {
+    if (input !== 'true') {
+      const packageManagers = input.split(' ');
```

Comment:
Maybe also trim the input and split by `\s+` to avoid issues due to leading, trailing and double spacing